wavefront/server/modules/product_analysis_module/tests/test_product_analysis_controllers.py

```
@@ -369,3 +371,170 @@ async def test_get_product_analysis_with_query_params(
+    included_user_email = 'included.user@example.com'
+    excluded_user_email = 'rootflo@muthootgroup.com'
+    zero_user_email = 'zero.user@example.com'
+
+    included_user_id = test_user_id
+
+    async with test_session() as session:
+        session.add(
+            User(
+                id=included_user_id,
+                email=included_user_email,
+                password='hashed_password',
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "password"

(S106)